gapis/service/service.proto

```diff
@@ -316,6 +316,16 @@ message DCECaptureResponse {
   }
 }
 
+message GraphVisualizationFileRequest {
```

Comment:
These RPCs will fail if the returned file exceeds 2GB. We should probably be careful to make sure that is a rare case.

If we think it will be larger, than as a follow-up CL, we should turn this into a streaming RPC. Does not have to be done now.

Reply:
I have push a new commit in response to changes requested . Please review .